crates/sui-indexer/src/lib.rs

```diff
@@ -179,19 +186,41 @@ pub fn establish_connection(db_url: String) -> PgConnection {
     PgConnection::establish(&db_url).unwrap_or_else(|_| panic!("Error connecting to {}", db_url))
 }
 
-pub fn new_pg_connection_pool(db_url: &str) -> Result<PgConnectionPool, IndexerError> {
+pub async fn new_pg_connection_pool(
+    db_url: &str,
+) -> Result<(PgConnectionPool, AsyncPgConnectionPool), IndexerError> {
     let manager = ConnectionManager::<PgConnection>::new(db_url);
     // default connection pool max size is 10
-    Pool::builder().build(manager).map_err(|e| {
+    let blocking_cp = diesel::r2d2::Pool::builder().build(manager).map_err(|e| {
         IndexerError::PgConnectionPoolInitError(format!(
             "Failed to initialize connection pool with error: {:?}",
             e
         ))
-    })
+    })?;
+
+    let manager = AsyncDieselConnectionManager::<AsyncPgConnection>::new(db_url);
+
+    let connection_size = env::var("DB_CONNECTION_SIZE")
+        .unwrap_or_else(|_| "100".to_string())
+        .parse::<u32>()
+        .unwrap_or(100);
```

Comment:
Our vultr instances allow up to 197 concurrent connections, can we default to that number instead?